pkg/secrets/filedriver/filedriver.go

```diff
@@ -79,7 +79,7 @@ func (d *Driver) Lookup(id string) ([]byte, error) {
 }
 
 // Store stores the bytes associated with an ID. An error is returned if the ID arleady exists
-func (d *Driver) Store(id string, data []byte) error {
+func (d *Driver) Store(id string, data map[string][]byte) error {
```

Comment:
I think Store() should not be storing map[string][]byte, but rather still just []byte. In the eyes of podman, a kube secret that looks like 
```
apiVersion: v1
kind: Secret
metadata:
  name: mysecret
type: Opaque
data:
  USER_NAME: YWRtaW4=
  PASSWORD: MWYyZDFlMmU2N2Rm
```

This ENTIRE chunk should be regarded as one secret, not just the USER_NAME and the PASSWORD. 

Only on retrieval, would the secret be regarded as USER_NAME and PASSWORD.